contact/obs.js

```diff
@@ -39,6 +41,18 @@ exports.create = function (api) {
             }
           })
         }
+        if (typeof msg.value.content.blocking === 'boolean') {
+          update(source, {
```

Comment:
oh wait, i just realized: if both the following code and the blocking code update `source` and `target` with an object `{ following: { ... } }` or `{ blocking: { ... } }`, will they overwrite each other? if so, is that okay? i wonder if we need to update with a merge (update only the relevant key `following` or `blocking`, not the entire object).

Reply:
Good questions. This was at the end of a bunch of digging (there are some dark dead code areas I wandered through). I trusted the person who wrote this to be doing a merge because it looked like they were looping over keys !